src/components/NewBookingModal/NewBooking/form.js

```diff
@@ -95,85 +95,59 @@ class RawForm extends Component {
 
   renderErrors = errors => {
     return (
-      <ul>
+      <FormValidationErrorMessage>
         {Object.values(errors).map((error, index) => (
           <li key={Object.keys(errors)[index]}>{error}</li>
         ))}
-      </ul>
+      </FormValidationErrorMessage>
     );
   };
 
-  dateSelectionChanged = value => {
-    const { values, setFieldValue } = this.props;
-    if (values.endDate.isBefore(value)) {
-      setFieldValue('startDate', value);
-      setFieldValue('endDate', value);
-    } else if (values.isHalfDay) {
-      setFieldValue('endDate', value);
-      setFieldValue('startDate', value);
-    } else {
-      setFieldValue('startDate', value);
-    }
-  };
-
   render() {
     const {
+      eventTypes,
       values,
-      touched,
       errors,
-      handleChange,
-      handleSubmit,
-      setFieldValue,
     } = this.props;
 
     return (
-      <form onSubmit={handleSubmit}>
-        <label htmlFor="eventTypeId">Booking Type</label>
-        <select
-          id="eventTypeId"
-          onChange={handleChange}
-          value={values.eventTypeId}
-          className={errors.eventTypeId && touched.eventTypeId ? 'error' : ''}
-        >
-          <option value={1}>Annual Leave</option>
-          <option value={2}>Working From Home</option>
-        </select>
-        <label htmlFor="startDate">Start Date</label>
-        <DatePicker
-          id="startDate"
-          selected={values.startDate}
-          onChange={value => this.dateSelectionChanged(value)}
-          className={errors.startDate && touched.startDate ? 'error' : ''}
+      <Form>
+
+        <Field
```

Comment:
Just a really small thing, but this kind of spacing between the form and field components is a bit weird. Noticed it in other PRs too.

Reply:
Yeah, I just do this for easier legibility. Its just a preference